openstates/get-state-lawmakers.js

```diff
@@ -187,28 +135,77 @@ async function getStateLegs() {
       .set('X-API-Key', OpenStatesAPIKey)
       .send(generateOpenStatesQueryString(stateName))
       .then((data) => {
+        console.log('got data')
         return transformOpenStatesLegislatorsData(data.body);
       })
       .then((lawmakers) => {
         Object.keys(lawmakers).forEach(memberId => {
-          const member = lawmakers[memberId];
-          member.thp_id = memberId;
-          checkIsInDb(member)
-            .then((isInDatabase) => {
-              if (isInDatabase) {
-                console.log('already there, updating', member.thp_id)
-                update(member);
+          const person = lawmakers[memberId]
+          const newOfficePerson = new StateLawmaker(memberId, person.openStatesDisplayName, person.state)
```

Comment:
to begin with, a 'handleOfficePerson(newOfficePerson)' function would be good to call here.